README.md

```diff
@@ -110,30 +110,29 @@ hyperfine -N 'grep TODO /home/user'
 
 ### Shell functions and aliases
 
-If you are using bash, you can export shell functions to directly benchmark them with hyperfine:
+Bash supports saving functions through the environment:
```

Comment:
Why do you think this is better?

Reply:
Impersonal, explicitly lists what the mechanism is (you can't "export functions directly", because you export to the environment block which contains strings, not functions), and explicitly notes that this requires bash on both sides.